code/modules/library/book.dm

```diff
@@ -177,9 +177,10 @@
 		return FALSE
 	if(!user.combat_mode)
 		return FALSE
-	if(!((carving_item.sharpness & SHARP_EDGED) && (carving_item.tool_behaviour != TOOL_KNIFE) && (carving_item.tool_behaviour != TOOL_WIRECUTTER)))
+	//special check for wirecutter's because they don't have a sharp edge
+	if((carving_item.tool_behaviour != TOOL_WIRECUTTER) && !(carving_item.sharpness & SHARP_EDGED))
```

Comment:
```suggestion
	if((carving_item.sharpness & SHARP_EDGED) || (carving_item.tool_behaviour == TOOL_WIRECUTTER))
```

even more readable

Reply:
Interesting

Reply:
The if condition broke the fix since i early return on the next line. Have to move everything else after this inside